examples/sklearnex/random_forest_regressor_dpnp.py

```diff
@@ -18,13 +18,23 @@
 #    python ./random_forest_regressor_dpnp_batch.py
 
 import dpnp
-import numpy as np
 from sklearn.datasets import make_regression
 from sklearn.model_selection import train_test_split
 
+# Just directly import estimator from sklearnex
 from sklearnex.ensemble import RandomForestRegressor
 
-sycl_device = "gpu:0"
+## Or import estimator via sklearnex's patch mechanism from sklearn
```

Comment:
Similar comment as in the other file, rework into sentences, try to make it in a single block.